test/t8_cmesh/t8_gtest_cmesh_copy.cxx

```diff
@@ -22,57 +22,66 @@
 
 #include <gtest/gtest.h>
 #include <t8_cmesh.h>
+#include <t8_schemes/t8_default/t8_default_cxx.hxx>
 #include "t8_cmesh/t8_cmesh_trees.h"
 #include "t8_cmesh/t8_cmesh_partition.h"
-#include <t8_eclass.h>
 #include <t8_cmesh/t8_cmesh_testcases.h>
 #include <test/t8_gtest_macros.hxx>
 
-/* Test if a cmesh is committed properly and perform the face consistency check. */
+/* We create and commit a cmesh, then derive a new cmesh
+ * from it without any changes.
+ * We test whether the new and original cmesh are equal.
+ */
 
-class cmesh_copy_equality: public testing::TestWithParam<int> {
+/* Note: This test currently fails on many cmeshes and is thus deavtivated.
+ * See: https://github.com/DLR-AMR/t8code/issues/920*/
+* /
+
+  class t8_cmesh_copy: public testing::TestWithParam<int> {
  protected:
   void
   SetUp () override
   {
+    /* Skip test since cmesh copy is not yet working. See https://github.com/DLR-AMR/t8code/issues/920 */
+    GTEST_SKIP ();
```

Comment:
Why call `GTEST_SKIP` here not just in the `TEST_P` body?

Reply:
B/c the we get one skipped test output for each parameter, ie 160 lines of "test .... skipped", which clutters the output.
Instead, now we get only one line of "test suite ... skipped".

Need to do it in teardown as well to avoid memory issues.